src/coreclr/tools/aot/ILCompiler.LLVM/CodeGen/ILToLLVMImporter_Statics.cs

```diff
@@ -379,6 +379,40 @@ public static int PadOffset(TypeDesc type, int atOffset)
             return padding;
         }
 
+        public static void GetObjectLayoutInstructions(TypeDesc type, List<FieldStoreLayout> layout, int startOffset)
+        {
+            foreach (FieldDesc f in type.GetFields())
+            {
+                if (f.IsStatic) continue;
```

Comment:
```suggestion
                if (f.IsStatic)
                {
                    continue;
                }

```
(Or without the braces if that's the prevailing style)